lib/app_component.dart

```diff
@@ -380,7 +379,7 @@ class AppComponent {
       equationItems.add(totalValue);
 
       //this fixes precision issues sqrt(2)*sqrt(2) = 2.000000000004
-      if (totalValue.toString().contains(".")) {
+      if (totalValue.toString().contains(decimal)) {
```

Comment:
This was actually referenced [here](https://github.com/DocJava/Dartulator/pull/7). doing an "is double" check was not working even when Dart was set to strong typed mode. x.truncate() == x did not work either.  This is not an ideal fix and once a more promising fix comes along I will be moving over to it.